### PR TITLE
[TRANSLATIONS]: Malayalam 

### PR DESCRIPTION
**Pull request by varghesejose2020**
help needed to translated to Readme to malayalam

### CODE REVIEW

**Review by jlooper — COMMENTED** [review]
Hi, do you want to convert this to draft and work on it progressively?

**Review by Microsoft-code — APPROVED** [review]

File: translations/README.ml.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+ഇതുകൂടാതെ, ഒരു ക്ലാസിന് മുമ്പുള്ള ഒരു കുറഞ്ഞ ക്വിസ് ഒരു വിഷയം പഠിക്കുന്നതിനുള്ള വിദ്യാർത്ഥിയുടെ ഉദ്ദേശ്യം സജ്ജമാക്കുന്നു, അതേസമയം ക്ലാസിന് ശേഷമുള്ള രണ്ടാമത്തെ ക്വിസ് കൂടുതൽ നിലനിർത്തൽ ഉറപ്പാക്കുന്നു. ഈ പാഠ്യപദ്ധതി രൂപകൽപ്പന ചെയ്തിരിക്കുന്നത് വഴക്കമുള്ളതും രസകരവുമാണ്, ഇത് മുഴുവനായോ ഭാഗികമായോ എടുക്കാം. പദ്ധതികൾ ചെറുതായി ആരംഭിച്ച് 12 ആഴ്ച ചക്രത്തിന്റെ അവസാനത്തോടെ കൂടുതൽ സങ്കീർണമാകുന്നു.
+
+ഒരു ചട്ടക്കൂട് സ്വീകരിക്കുന്നതിന് മുമ്പ് ഒരു വെബ് ഡെവലപ്പർ എന്ന നിലയിൽ ആവശ്യമായ അടിസ്ഥാന കഴിവുകളിൽ ശ്രദ്ധ കേന്ദ്രീകരിക്കുന്നതിന് ഞങ്ങൾ ജാവാസ്ക്രിപ്റ്റ് ചട്ടക്കൂടുകൾ അവതരിപ്പിക്കുന്നത് മനപ്പൂർവ്വം ഒഴിവാക്കിയിട്ടുണ്ടെങ്കിലും, ഈ പാഠ്യപദ്ധതി പൂർത്തിയാക്കുന്നതിനുള്ള ഒരു നല്ല അടുത്ത ഘട്ടം വീഡിയോകളുടെ മറ്റൊരു ശേഖരത്തിലൂടെ Node.js- നെക്കുറിച്ച് പഠിക്കുക എന്നതാണ്: "[Beginner Series to: Node.js](https://channel9.msdn.com/Series/Beginners-Series-to-Nodejs?WT.mc_id=academic-13441-cxa)".
+
+> Find our [Code of Conduct](CODE_OF_CONDUCT.md), [Contributing](CONTRIBUTING.md), and [Translation](TRANSLATIONS.md) guidelines. We welcome your constructive feedback!
```
[the remark this answers is not in the document]
Comment (Justinjdaniel) [reply translations/README.ml.md:39]: If you don't mind, I can also help out with the remaining English strings.

**Comment by Justinjdaniel** [comment]
> yes ,need help from other contributors work on progress @jlooper

Happy to help

File: translations/README.ml.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (varghesejose2020) [reply translations/README.ml.md:39]: Yes Proceed @Justinjdaniel

[the remark this answers is not in the document]
Comment (varghesejose2020) [reply translations/README.ml.md:39]: If there are any errors in my translation, you can fix them @Justinjdaniel

[the remark this answers is not in the document]
Comment (Justinjdaniel) [reply translations/README.ml.md:39]: @varghesejose2020 Sure 👍🏻

**Comment by Justinjdaniel** [comment]
**Done Translating** 
how do I push my changes to this pull request when this one is drafted? 🤔
Should I go for a new Pull Request?

**Comment by jlooper** [comment]
> **Done Translating** how do I push my changes to this pull request when this one is drafted? 🤔 Should I go for a new Pull Request?

hi, you can add more commits to your branch and the PR will be updated, thanks. When it's no longer a draft, you can ask for a review and move it from status 'draft'

**Comment by varghesejose2020** [comment]
Can a new contributor edit or add more to this PR of mine? @jlooper

**Comment by varghesejose2020** [comment]
#510

**Comment by jlooper** [comment]
> #510 already merged
> 
> please close #427

Closing this one, then. Thank you all!

**Pull request closed without merging**